.circleci/config.yml

```diff
@@ -99,19 +99,30 @@ jobs:
     docker:
       - image: cimg/node:16.15-browsers
     steps:
-      - checkout
-      - browser-tools/install-chrome
-      - browser-tools/install-chromedriver
-
-      #install-packages documentation: https://circleci.com/developer/orbs/orb/circleci/node#commands-install-packages
-      - node/install-packages:
-          override-ci-command: npm install
-          app-dir: ~/project/front-end/
-
+      - setup_remote_docker:
+          docker_layer_caching: true
       - run:
-          name: run e2e tests
-          command: node --max_old_space_size=4000 ./node_modules/@angular/cli/bin/ng e2e
-          working_directory: ~/project/front-end/
+          name: init fecfile-web-api
```

Comment:
This step does a lot more than init fecfile-web-api.  In a lot of cases (obviously not always), I think more steps means more clarity when we look at what failed or what's slow.  plus, circle ci has a lot of steps already made like `-checkout`

It might be worth splitting these up into `initialize api`, `-checkout`(for the front end), `- node/install-packages` (again for the front end), and `run e2e tests`

Reply:
like here, the api was initialized fine and didn't take an hour, but it looks like it failed and took forever to build
![image](https://user-images.githubusercontent.com/97105825/205991597-6d2eb125-098a-42bb-9ea1-1fb5604e8f25.png)